File: src/main/java/it/usuratonkachi/telegranloader/telegranloader/TelegranloaderApplication.java
package it.usuratonkachi.telegranloader.telegranloader;

import it.usuratonkachi.telegranloader.telegranloader.bot.TelegranLoaderProperties;
import it.usuratonkachi.telegranloader.telegranloader.bot.TelegranLoaderWebhook;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.context.properties.EnableConfigurationProperties;
import org.telegram.telegrambots.ApiContextInitializer;

@SpringBootApplication
@EnableConfigurationProperties(value = TelegranLoaderProperties.class)
public class TelegranloaderApplication {

	public static void main(String[] args) {
		ApiContextInitializer.init();
		SpringApplication.run(TelegranloaderApplication.class, args);
	}

}

File: src/main/kotlin/it/usuratonkachi/telegranloader/TelegranloaderApplication.kt
package it.usuratonkachi.telegranloader

import it.tdlight.common.Init
import it.usuratonkachi.telegranloader.bot.TelegramBotProperties
import it.usuratonkachi.telegranloader.config.TelegramApiProperties
import it.usuratonkachi.telegranloader.config.TelegramCommonProperties
import it.usuratonkachi.telegranloader.parser.ParserRefactorConfiguration
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.context.properties.EnableConfigurationProperties
import org.springframework.boot.runApplication


@SpringBootApplication
@EnableConfigurationProperties(TelegramCommonProperties::class, TelegramApiProperties::class, TelegramBotProperties::class, ParserRefactorConfiguration::class)
class TelegranloaderApplication

fun main(args: Array<String>) {
	Init.start()
	runApplication<TelegranloaderApplication>(*args)
}
